Library/Formula/chapel.rb

```diff
@@ -27,8 +27,9 @@ def install
     prefix.install_metafiles
 
     # Install chpl and other binaries (e.g. chpldoc) into bin/ as exec scripts.
-    bin.install Dir[libexec/"bin/darwin/*"]
-    bin.env_script_all_files libexec/"bin/darwin/", :CHPL_HOME => libexec
+    platform = "#{OS::NAME}#{OS.linux? ? Hardware::CPU.bits.to_s : ""}"
```

Comment:
The `.to_s` is necessary, unfortunately.

I'll try `#{Hardware::CPU.bits if OS.linux?}`. Thanks a bunch for the suggestion - I really tried my hardest to make this concise, but I'm new to Ruby (I don't use it for any other project) so it can be challenging sometimes.